### DISCUSSION TITLE
golang/%E6%A0%B8%E5%BF%83%E7%9F%A5%E8%AF%86%E7%82%B9/golang%E5%B9%B6%E5%8F%91%E9%A2%98%E7%9B%AE%E6%B5%8B%E8%AF%95

### DISCUSSION THREAD

**Discussion by giscus** [xiaobaiTech/golang-full-stack#12] [Announcements]
# golang/%E6%A0%B8%E5%BF%83%E7%9F%A5%E8%AF%86%E7%82%B9/golang%E5%B9%B6%E5%8F%91%E9%A2%98%E7%9B%AE%E6%B5%8B%E8%AF%95

golang 并发题目测试 题目来源： Go并发编程小测验： 你能答对几道题？ 1 Mutex A: 不能编译 B: 输出 main --> A --> B --> C C: 输出 main D: panic 2 RWMutex A: 不能编译 B: 输出 1 C: 程序hang住 D: panic 3 Waitgroup A: 不能编译 B: 无输出...

https://golangguide.top/golang/%E6%A0%B8%E5%BF%83%E7%9F%A5%E8%AF%86%E7%82%B9/golang%E5%B9%B6%E5%8F%91%E9%A2%98%E7%9B%AE%E6%B5%8B%E8%AF%95.html

**Comment by kuooyoung** [comment]
第四题不对吧，应该不会重复执行？
